sdk/tests/conformance2/rendering/attrib-type-match.html

```diff
@@ -115,11 +216,8 @@
     }
     wtu.glErrorShouldBe(gl, gl.NO_ERROR, "No GL error from set up");
 
-    debug("");
-    debug("Test vertexAttrib with drawArrays and drawArraysInstanced");
     wtu.setupUnitQuad(gl, 0);
 
-    debug("Correct setup");
```

Comment:
Maybe "correct setup" isn't clear enough.  It probably should say "draw call should succeed with correct setup."